### DISCUSSION TITLE
Use a list of modified dates in blog posts instead of a single date.

### DISCUSSION THREAD

**Discussion by e-alizadeh** [quarto-dev/quarto-cli#5357] [Feature Requests]
It would be very nice if the `date-modified` field accepts a list of dates instead of a single date. 
The reason I'm interested in this is that I may update my blog posts a couple of times over the years and it would be nice to see the list of all dates the post was updated.

**Comment by mcanouil** [comment]
I personally don't see the rational to see all dates of modifications, especially if you are using Git and your post is linked to GitHub for example.

Could you elaborate on the goal and needs for such feature?

**Comment by e-alizadeh** [comment]
@mcanouil The reason behind seeing all dates of modifications is not for myself but for an intended reader. My reasons for suggesting such a feature is for a reader to see the number of times and the dates the post is updated (the git repo can be private). 

 On a related note, does it make sense to have the ability to add a custom field to the blog description? 
This would make it much more customizable, and if someone (like me) wants to add something that's not supported, (s)he can use this field.

**Reply by mcanouil** [reply]
I still don't see the point of having the number of times a document has been modified if you actually do not have the changes made. It does not matter for whom it is for.
What matters is what would be the goal and use for such feature? (I am not saying, there is no need for this or that there is one either).

"Description" is a field, so not sure what you are referring to, but see "custom listings"
(<https://quarto.org/docs/websites/website-listings-custom.html>) and "partials" (<https://quarto.org/docs/journals/templates.html#html-partials>).

**Comment by bryanhanson** [comment]
Take a look at "Update History" at the bottom of the first page of this document by [Frank Harrell](https://hbiostat.org/bbr/) for another way to tell your readers about document changes.

**Reply by mcanouil** [reply]
Indeed, thanks for this example which is in my opinion better than a list of dates (in some kind of dropdown menu), because it brings context.
To note, it's a simple callout and markdown table.

```qmd
::: {.callout-note collapse="true"}
# Update History

| Date | Sections | Changes | Thanks To |
|:-----|:---------|:--------|:----------|
| 2023-04-27 | [-@sec-nonpar-ecdf]      | New section on sample size for ECDF |
| 2023-04-05 | [-@sec-change-ex]        | Added confidence bands |
| 2023-03-30 | [-@sec-hdata-simor]      | Fixed bug in simulation graphics |
| 2023-03-29 | [-@sec-overview-scimeth] | New link to clinical trial design resource |
| 2023-03-13 | [-@sec-repro] | New subsection on the decline effect |
| 2023-02-19 | [-@sec-prob]  | Added link to resources for learning probability |
| 2022-12-29 | [-@sec-descript-models]| Added single-axis nomogram example |
| 2022-12-28 |                        | Started to add old study questions to end of selected chapters |
| 2022-12-03 | [-@sec-change-ex]      | New section with real example of misleading change score |
| 2022-11-27 | [-@sec-change-current] | New section on importance of current status vs. baseline status and irrelevance of change for patients |
| 2022-08-02 | [-@sec-dx]             | Quote about weaknesses in sens and spec; link to CrossValidated discussion | |
| 2022-08-31 | [-@sec-corr-n]         | New material on sample size vs. P(correct sign on r) | |

:::
```

**Reply by e-alizadeh** [reply]
Thanks @bryanhanson @mcanouil. This is actually better than what I was looking for!
